storage/ndb/rest-server/data-access-rondb/src/db-operations/pk/common.cpp

```diff
@@ -312,23 +291,24 @@ RS_Status SetOperationPKCol(const NdbDictionary::Column *col, NdbOperation *oper
   case NdbDictionary::Column::Decimal: {
```

Comment:
A generic comment is that if you are ready to return from function with RS_OK, do it immediately without adding one more if statement and one more variable. This is replicated in many case-statements

Reply:
One more generic comment, if possible ensure that the common path is that the if-condition is true. If not possible try to use unlikely.

Reply:
One more generic comment: NEVER, EVER use try-catch in C++, it is a performance bug and complicates simple code to make it simple. Was one of my first lessons in my career, try-catch can cause unsolvable issues.